recipes/loos/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set version = "3.2" %}
+
+package:
+  name: "loos"
+  version: "{{ version }}"
+
+source:
+  url: https://raw.githubusercontent.com/GrossfieldLab/loos/master/archive/loos-{{ version }}.tar.gz
+  sha256: 4d3806e712e4ae2190385db15773a1af2319dc1c7774e0d44f51370a9061bfbc
+
+build:
+  number: 0
+  # Only python 3.x, no windows
+  skip: True  # [win or py<35]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - scons
+    - numpy
+    - boost
+    - boost-cpp
+    - openblas
+    - libnetcdf
+    - eigen
```

Comment:
```suggestion
```

Reply:
I understand what you're suggesting (it's what I had originally, and seems more consistent with the instructions), but when I had them only in host and not in build, the install couldn't find them.

Reply:
That's probably because you had `PREFIX` wrong

Reply:
I don't think so -- the build was failing long before PREFIX gets used (that's literally ONLY at install, not during the build). However, I'll check to be sure